packaging/docker/build.sh

```diff
@@ -1,82 +1,56 @@
 #!/bin/bash
```

Comment:
After this split, this file and `build-test.sh` are duplicating functions.

Reply:
Will integrate them later.
For now it seems that build-test is used for manual testing, while build.sh is used with Travis.
The obvious solution is to make build-test.sh call build.sh after setting up all required variables.

I will do this with a separate commit, for now i will just add a TODO on build-test.sh so that we dont forget